rock_paper_scissors/api/models.py

```diff
@@ -0,0 +1,65 @@
+"""
+Relationships with Foreign Keys, esp. relationship() and FK values
+Why do we need to establish columns for players in game?
+difference with many to one and one to many -- if you back_populate either relationship type,
+    does it matter which you use?
+
+"""
+
+from sqlalchemy import Column, Integer, String
+from sqlalchemy import Enum
+from sqlalchemy import ForeignKey
+from sqlalchemy.ext.declarative import as_declarative
+from sqlalchemy.orm import relationship
+
+
+@as_declarative()
+class ModelBase(object):
+    def __tablename__(self):
+        return self.__class__.__name__.lower()
+
+
+class Game(ModelBase):
+    id = Column(Integer, primary_key=True)
+    player_1_id = Column(Integer, ForeignKey('user.id'))
+    player_2_id = Column(Integer, ForeignKey('user.id'))
+    round_id = Column(Integer, ForeignKey('round.id'))
+    rounds_choice = Column(Integer)
+    rounds_decrement = Column(Integer)
+    current_round = Column(Integer)
+    winner = Column(String, ForeignKey('user.id'))
+
+    round = relationship('Round', back_populates='game')
+    user = relationship('User', back_populates='game', uselist=False)
```

Comment:
@TylerHendrickson this `Dog` to `Breed` explanation was very helpful. Basically the relationships api was built into SQLAchemy for the benefit of this direct model to model relationship is that correct? As I've been going over the bullpen codebase, specifically the models, a lot of the models have a relationship setup between other models. Is this 'optional' feature usually implemented because of its' handiness?